recipes/limereport/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.1)
```

```suggestion
cmake_minimum_required(VERSION 3.15)
```

[Compatibility with CMake < 3.5 will be removed from a future version of CMake](https://cmake.org/cmake/help/latest/release/3.27.html#deprecated-and-removed-features), let's try to stick to 3.15 for tests :)

---

done